nixos/modules/services/x11/xserver.nix

```diff
@@ -655,7 +662,7 @@ in
           ${cfgPath}.source = xorg.xf86inputevdev.out + "/share" + cfgPath;
         });
 
-    environment.systemPackages =
+    environment.systemPackages = pkgs.gnome.removePackagesByName
```

Comment:
Move this to somewhere else? Seems like strange layering to pull in functions from `gnome` for xserver.  Would it make sense somewhere in `lib`?

Reply:
I moved it to nixos/utils for now.